src/main/java/org/libertybikes/ai/service/RegistrationBean.java

```diff
@@ -27,30 +25,22 @@
 
     private static final Jsonb jsonb = JsonbBuilder.create();
 
-    @Inject
     AIConfiguration config;
 
-    @Inject
-    @RestClient
     GameServiceClient gameService;
 
     public void joinRound() {
         System.out.println("Attempting to register with game service...");
 
-        String partyId = (String) gameService.describe().get("partyId");
-        System.out.println("Found party id: " + partyId);
+        // TODO: use MPRestClient to pull the valid partyID from the game service
+        String partyId = "";
 
         // MP Rest Client doesn't yet support JAX-RS server-sent-events, so we nee to do this manually
         // enhancement tracked at: https://github.com/eclipse/microprofile-rest-client/issues/11
         String targetStr = config.getGameServiceHttp() + "/party/" + partyId + "/queue?playerId=" + config.getPlayerId();
```

Comment:
lets add a check in here like:
```java
if (config == null) {
  System.out.println("Injection for AIConfiguration not yet implemented!");
  return;
}
```
so people don't hit NPEs when they are following the correct steps in the lab